src/interfaces.jl

```diff
@@ -472,8 +472,7 @@ end
 """
     `plottype(plot_args...)`
 
-The default plot type for any argument is `lines`.
-Any custom argument combination that has only one meaningful way to be plotted should overload this.
```

Comment:
I changed the wording. IMO, as long as one method is _preferred_, one should overload this. The user can force their preference regardless, so we don't lose anything by having a sensible default.